hathor/merged_mining/coordinator.py

```diff
@@ -71,6 +71,9 @@
 # PROPAGATION_FAILED = {'code': 33, 'message': 'Solution propagation failed'}
 DUPLICATE_SOLUTION = {'code': 34, 'message': 'Solution already submitted'}
 
+ZEROED_4: bytes = b'\0' * 4
+ZEROED_32: bytes = b'\0' * 32
+
 
 class HathorCoordJob(NamedTuple):
```

Comment:
I guess that works, but I'd have to make a protocol or abstract base class to ensure type safety.

Reply:
Oh, wait, don't you mean `EmptyBitcoinCoordJob` or `DummyBitcoinCoordJob`? Because a `HathorCoordJob` is the job that will be submitted to the Hathor node when possible and this PR doesn't change that, the dummy merged mining still submits the results to the Hathor node, it just doesn't use a Bitcoin node.